nft_ingester/src/api/error.rs

```diff
@@ -95,6 +97,11 @@ impl From<DasApiError> for jsonrpc_core::Error {
                     .to_string(),
                 data: None,
             },
+            DasApiError::QueryTimedOut => jsonrpc_core::Error {
+                code: ErrorCode::InvalidRequest,
```

Comment:
Probably it's more of an InternalError, or rather ServerError with some custom code. At least I've seen from some other providers
```
"error": {
    "code": -32504,
    "message": "Request timed out, please try again."
  }
```

Reply:
I see this error more as a badly written query that takes longer than it should. Therefore, in my opinion, this error is not related to our server, but to the specifics of the client’s request, who wrote an inappropriate request.